instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/internal/JdbcConnectionUrlParser.java

```diff
@@ -831,6 +831,54 @@ DbInfo.Builder doParse(String jdbcUrl, DbInfo.Builder builder) {
 
       return MODIFIED_URL_LIKE.doParse(jdbcUrl, builder);
     }
+  },
+  INFORMIX_SQLI("informix-sqli") {
+    @Override
+    DbInfo.Builder doParse(String jdbcUrl, DbInfo.Builder builder) {
+      builder = MODIFIED_URL_LIKE.doParse(jdbcUrl, builder);
+
+      int dbNameStartIdx = jdbcUrl.indexOf('/', jdbcUrl.indexOf("//") + 2) + 1;
+      int dbNameEndIdx = jdbcUrl.indexOf(':', dbNameStartIdx);
+
+      String name = jdbcUrl.substring(dbNameStartIdx, dbNameEndIdx);
```

Comment:
will this work if you try to parse `jdbc:informix-sqli://infxhost:99`

Reply:
i added these two variations as test cases and updated the logic